lib/tldr.js

```diff
@@ -140,15 +145,16 @@ function printBestPage(command, options={}) {
         return exit();
       }
       // If not found, try to update
-      console.log('Page not found. Updating cache ..');
+      spinner.start('Page not found, updating cache');
```

Comment:
I would prefer to keep 2 sentences. I think grammatically that is more correct. You can remove the ellipses (`...`)

Reply:
`ora` style is without any suffix so I tried to go with that which made me change from period to colon. :grin:
So you ar basicly saying lets use:
`Page not found. Updating cache`?

Reply:
Correct.